docs/learn/deploy/3b-expfactory.rst

```diff
@@ -0,0 +1,141 @@
+.. _tutorial/deploy/third-party/expfactory:
+
+The Experiment Factory
```

Comment:
Ok, totes love this! I'd like to suggest some largely cosmetic changes (link fixes/fights with ReST, and consistent naming of `lab.js`), and for the `lab.js` docs, I think it would be cool to add some context about expfactory:

```
Reproducible studies with the Experiment Factory
================================================

`The Experiment Factory`_, or "expfactory" for short, is is an open source infrastructure for building and deploying reproducible experiment containers. You can export a ``lab.js`` experiment and build it into a container (or combine multiple experiments, even if they are built with different tools) that is ready for deployment on a webserver with technologies like SSL, and data collection options ranging from flat files to `databases <https://expfactory.github.io/expfactory/customize#database>`_ (Sqlite, Postgres and MySQL).

.. seealso::

  The ambition and the powers of `the Experiment Factory`_ go *way beyond* hosting ``lab.js`` studies. Please also check out:

  * Vanessa Sochat's (et al.) papers outlining the vision of `standardizing experiments <https://www.frontiersin.org/articles/10.3389/fpsyg.2016.00610/full>`_ and using `reproducible experiment containers <http://joss.theoj.org/papers/10.21105/joss.00521>`_.
  * Their extensive `library of pre-made paradigms <https://expfactory.github.io/experiments/>`_, all of which are free to use.
  * Their `documentation on integrating <https://expfactory.github.io/expfactory/integration-labjs>`_ with ``lab.js``

.. _The Experiment Factory: https://expfactory.github.io

.. contents:: Steps
  :local:
```

Also, I really dig the database support! Will check that out sortly.